corehq/apps/app_manager/models.py

```diff
@@ -2520,7 +2520,14 @@ def get_case_updates(self, case_type):
             if action.case_type == case_type:
                 updates.update(format_key(*item)
                                for item in action.case_properties.iteritems())
-        return updates
+        if self.source:
+            xform = self.wrapped_xform()
```

Comment:
Should check if scheduler is enabled for this form before doing this since it's a reasonably expensive operation.